Cargo.toml

```diff
@@ -4,6 +4,13 @@ version = "0.1.0"
 authors = ["mlange-42 <martin_lange_@gmx.net>"]
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[bin]]
+name = "git-graph"
+path = "src/main.rs"
+
+[lib]
+path = "src/lib.rs"
 
 [dependencies]
+git2 = "0.13"
+time = "0.1"
```

Comment:
The `time` crate is in a bit of an odd place, `chrono` is probably a better place to look for calendar-based time handling.

Reply:
Removed print_time (incl. dependency) and other unused stuff / dead code.